nbconvert/preprocessors/tests/test_execute.py

```diff
@@ -40,7 +42,64 @@ def _normalize_base64(b64_text):
     except (ValueError, TypeError):
         return b64_text
 
-class TestExecute(PreprocessorTestsBase):
+
+def merge_dicts(first, second):
+    # Because this is annoying to do inline
+    outcome = {}
+    outcome.update(first)
+    outcome.update(second)
+    return outcome
+
+
+class ExecuteTestBase(PreprocessorTestsBase):
+    def build_preprocessor(self, opts):
+        """Make an instance of a preprocessor"""
+        preprocessor = ExecutePreprocessor()
+        preprocessor.enabled = True
+        for opt in opts:
+            setattr(preprocessor, opt, opts[opt])
+        # Perform some state setup that should probably be in the init
```

Comment:
Is this a suggestion for a change that should be made in the ExecutePreprocessor `__init__` or a test specific init?